pkg/antctl/raw/multicluster/deploy/leader_cluster.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deploy
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+type leaderClusterOptions struct {
+	namespace     string
+	filename      string
+	antreaVersion string
+}
+
+var leaderClusterOpts *leaderClusterOptions
+
+var leaderClusterExamples = strings.Trim(`
+Create the leader cluster CRDs and deploy the "antrea-mc-controller" Deployment.
+$ antctl mc deploy leader-cluster --antrea-version v1.5.0 -n antrea-mcs
+
```

Comment:
We should add an example of using the pre-downloaded manifest.

Reply:
Done. Add an example..